### DISCUSSION TITLE
We are getting "EACCES: permission denied, mkdir '/root/.config/Cypress/cy/production" error message in Jenkins console.

### DISCUSSION THREAD

**Discussion by DeepakB01** [cypress-io/cypress#23752] [CI setup]
I am integrating cypress tests with Jenkins pipeline. I have used cypress/included:8.5.0 docker image
We are running  jenkins as non-root as we dont have access for root.

But we are getting following exception on Jenkins Console - 

A JavaScript error occurred in the main process
Uncaught Exception:
Error: EACCES: permission denied, mkdir '/root/.config/Cypress/cy/production'
    at Object.mkdirSync (fs.js:987:3)
    at Object.mkdirsSync (/root/.cache/Cypress/8.5.0/Cypress/resources/app/packages/server/node_modules/fs-extra/lib/mkdirs/mkdirs-sync.js:31:9)

Please help me resolve this issue.

**Comment by lmiller1990** [comment]
There is lots of resources about this, take a look:
- https://github.com/cypress-io/cypress-docker-images/issues/42
- https://github.com/cypress-io/cypress/issues/8283

I'm guessing it'll depend on your setup. I don't think this is a Cypress specific issue, but just a configuration issue (maybe solve via `chmod`, maybe something else...), so I'm not sure how much more I'll be able to help.
